lxd/resources/storage.go

```diff
@@ -276,6 +278,28 @@ func GetStorage() (*api.ResourcesStorage, error) {
 				}
 			}
 
+			// Get information about what's mounted.
+			mountInfo, err := os.ReadFile(procSelfMountInfo)
```

Comment:
@masnax am I reading this right? Is this code parsing `/proc/self/mountinfo` over and over for every block device in the system?

This doesn't seem very efficient. 
Can we process the list of block devices first, and then parse  `/proc/self/mountinfo` and for each entry look for a matching disk and update the `.Mounted` field as needed.

Reply:
Good catch, I just need to move this block up out if the loop. The logic isn't dependent on the loop at all.

Reply:
This should now be fixed.